### ISSUE TITLE
TipTacItemRef - Lua Errors with Wrath Classic 3.4.1

### ISSUE THREAD

**Issue by boktai1000** [frozn/TipTac#170]
**Describe the bug**
Lua Error:

```
1x TipTacItemRef/ttItemRef.lua:1859: hooksecurefunc(): CommunitiesGuildNewsButton_OnEnter is not a function
[string "=[C]"]: in function `hooksecurefunc'
[string "@TipTacItemRef/ttItemRef.lua"]:1859: in function `?'
[string "@TipTacItemRef/ttItemRef.lua"]:273: in function <TipTacItemRef/ttItemRef.lua:273>
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:385: in function `UIParentLoadAddOn'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:517: in function `Communities_LoadUI'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:687: in function `ToggleCommunitiesFrame'
[string "@FrameXML/FriendsFrame_Shared.lua"]:2424: in function `FriendsFrame_ToggleToCommunities'
[string "@FrameXML/FriendsFrame_Shared.lua"]:334: in function `FriendsFrame_Update'
[string "@FrameXML/FriendsFrame_Shared.lua"]:274: in function `FriendsFrame_OnShow'
[string "*FriendsFrame.xml:733_OnClick"]:2: in function <[string "*FriendsFrame.xml:733_OnClick"]:1>

Locals:
```

**To Reproduce**
Load game with TipTacItemRef enabled

**Expected behavior**
No Lua Errors

**Screenshots**

![image](https://user-images.githubusercontent.com/9054563/213015735-a8a47ea0-cd6a-46c0-b35a-47dad2469101.png)

**TipTac Reborn (please complete the following information):**
 - Version 22.12.15

**WoW (please complete the following information):**
 - Flavor: Wrath Classic
 - Version 3.4.1

**Additional context**
I'm currently using TipTacItemRef with the other modules disabled to get enhanced information on buffs such as Caster.

**Comment by frozn** [comment]
Thanks for reporting this. Will be fixed with the next release.

**Comment by frozn** [comment]
Fixed with release [v23.01.23](https://github.com/frozn/TipTac/releases/tag/v23.01.23-release).
